zebra-chain/Cargo.toml

```
@@ -33,7 +33,7 @@ hex = "0.4"
 incrementalmerkletree = "0.1.0"
 jubjub = "0.7.0"
 lazy_static = "1.4.0"
-orchard = { git = "https://github.com/zcash/orchard.git", rev = "2c8241f25b943aa05203eacf9905db117c69bd29" }
+orchard = "0.0"
```

Nit: Let's try to use full patch versions for all our dependencies, to avoid accidental duplicates.

```suggestion
orchard = "0.0.0"
```